src/Microsoft.Windows.Compatibility/src/README.md

```diff
@@ -1,9 +1,108 @@
-# Microsoft.Windows.Compatibility package
+# Microsoft.Windows.Compatibility
 
-This Windows Compatibility Pack provides access to APIs that were previously available only for .NET Framework. It can be used from both .NET as well as .NET Standard.
+The `Microsoft.Windows.Compatibility` package provides Windows-specific APIs to help you port your .NET Framework applications to .NET Core 2.0+, .NET 5+ or .NET Standard. This package offers a smoother transition for those looking to modernize their applications without losing access to familiar Windows functionalities.
 
-### How to add a new dependency
-1. Add the package version to the repository's Versions.props file.
-2. Add the package dependency to the repository's Version.Details.xml file. If the dependency doesn't come from a repository with a direct subscription to (i.e. winforms and runtime), add the `CoherentParentDependency` attribute to the dependency.
-3. Add the package reference into the Microsoft.Windows.Compatibility.csproj file and use the version just defined in the Versions.props file.
-4. If a `CoherentParentDependency` attribute was required, a dependency must be added to the repository dependency chain as well. I.e. to add a new dependency from runtime, a dependency must also be added in winforms and wpf's Version.Details.xml file.
\ No newline at end of file
+## Getting Started
+
+To start using the `Microsoft.Windows.Compatibility` package, you'll first need to install it via NuGet Package Manager, Package Manager Console, or by editing your project file.
+
+### NuGet Package Manager
+```
+Install-Package Microsoft.Windows.Compatibility -Version [your_version_here]
+```
+
+### .NET CLI
+```
+dotnet add package Microsoft.Windows.Compatibility --version [your_version_here]
+```
+
+### Prerequisites
+
+- .NET Core 2.0+
+- .NET 5+
+- .NET Standard 2.0 or higher
```

Comment:
Please remove these three sections. These are already part of the UI of nuget.org and tooling:

<img width="654" alt="image" src="https://github.com/dotnet/windowsdesktop/assets/7412651/eb793450-8fcf-4e53-8f3b-392fd084785f">

Reply:
That would then also count for `System.Drawing.Common`, I suppose, which did not have a readme, either?

Reply:
Did you already add a package readme for System.Drawing.Common? Yes, probably applies to that library as well.